helm-chart/crds/retrytimeoutconfig.yaml

```diff
@@ -0,0 +1,77 @@
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  name: retrytimeoutconfig.nsm.nginx.com
+  labels:
+    app.kubernetes.io/part-of: nginx-service-mesh
+spec:
+  group: specs.smi.nginx.com
+  scope: Namespaced
+  names:
+    kind: RetryTimeoutConfig
+    listKind: RetryTimeoutConfigList
+    shortNames:
+      - rtc
+    singular: retrytimeoutconfig
+    plural: retrytimeoutconfigs
+  versions:
+    - name: v1alpha1
+      served: true
+      storage: true
+      schema:
+        openAPIV3Schema:
+          type: object
+          required:
+            - spec
+          properties:
+            spec:
+              description: Retry and timeout configuration.
+              type: object
+              required:
+                - targetRef
+                - default
+              properties:
+                targetRef:
+                  type: object
+                  description: The resource this retry and/or timeout applies to.
+                  required:
+                    - name
+                    - kind
+                  properties:
+                    kind:
+                      description: Kind of the targetRef.
+                      type: string
+                      enum:
+                        - Service
+                    name:
+                      description: Name of the targetRef.
+                      type: string
+                      minLength: 1
+                    namespace:
+                      description: Namespace of the targetRef.
+                      type: string
+                    port:
+                      description: Port of the targetRef to apply the retry and/or timeout to if multiple ports are available.
+                default:
+                  description: Default retry and timeout values.
+                  type: object
+                  properties:
```

Comment:
Per the design doc, we should include `defaults` for each of these values that have one, for documentation purposes.